langchain4j-vertex-ai-gemini/src/test/java/dev/langchain4j/model/vertexai/VertexAiGeminiStreamingChatModelIT.java

```diff
@@ -34,16 +34,19 @@
 
 class VertexAiGeminiStreamingChatModelIT {
 
+    public static final String GEMINI_PRO = "gemini-1.5-pro-preview-0514";
+    public static final String GEMINI_PRO_VISION = "gemini-1.5-pro-preview-0514";
```

Comment:
is it intended that both constants refer to the same (gemini-1.5-pro-preview-0514) model?

Reply:
Good point, there's only really one model now, so there's no distinction between multimodal or not.
I'll update to just use one constant.

Reply:
Now using just one constant / single model.